Clarify notarization function (#59)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

2. Transparent Statement Consumers: entities, stakeholders, and users involved in supply chain interactions that need to access, validate, and trust authentic Statements.

Signed Statement Issuers rely on being discoverable and represented as the responsible parties for their registered Signed Statements via Transparency Services in a believable manner.

> The issuer of a Signed Statement should be authenticated and trusted according to the registration policy of the transparency service.

Author: ```suggestion
The issuer of a Signed Statement should be authenticated and authorized according to the registration policy of the transparency service.
```

Analogously, Transparent Statement Consumers rely on verifiable trustworthiness assertions associated with Transparent Statements and their processing provenance in a believable manner.
If trust can be put into the operations that record Signed Statements in a secure, append-only log via online operations, the same trust can be put into the resulting transparent statement,

> function) issued by the Transparency Services and that can be validated in offline operations.

Author: ```suggestion
issued by the Transparency Services and that can be validated in offline operations.
```

> If trust can be put into the operations that record Signed Statements in a secure, append-only Registry via online operations, the same trust can be put into a corresponding Receipt that is the resulting documentation (i.e transparent signed statement, via an endorsement from a notarization

Reviewer: Nit for consistency:
> append-only Registry 

Append-only Log

> If trust can be put into the operations that record Signed Statements in a secure, append-only Registry via online operations, the same trust can be put into a corresponding Receipt that is the resulting documentation (i.e transparent signed statement, via an endorsement from a notarization

Reviewer 2: ```suggestion
If trust can be put into the operations that record Signed Statements in a secure, append-only log via online operations, the same trust can be put into a corresponding Receipt that is the resulting documentation (i.e transparent signed statement, via an endorsement from a notarization
```

> If trust can be put into the operations that record Signed Statements in a secure, append-only log via online operations, the same trust can be put into a corresponding Receipt that is the resulting documentation (i.e transparent signed statement, via an endorsement from a notarization
> function) issued by the Transparency Services and that can be validated in offline operations.

Reviewer 2: ```suggestion
If trust can be put into the operations that record Signed Statements in a secure, append-only log via online operations, the same trust can be put into a corresponding Receipt that is the resulting documentation (i.e, transparent statement) issued by the Transparency Services and that can be validated in offline operations.
```

2. Transparent Statement Consumers: entities, stakeholders, and users involved in supply chain interactions that need to access, validate, and trust authentic Statements.

Signed Statement Issuers rely on being discoverable and represented as the responsible parties for their registered Signed Statements via Transparency Services in a believable manner.

> The issuer of a Signed Statement should be authenticated and trusted according to a registration policy of the transprency service.

Author: ```suggestion
The issuer of a Signed Statement should be authenticated and trusted according to the registration policy of the transparency service.
```

Analogously, Transparent Statement Consumers rely on verifiable trustworthiness assertions associated with Transparent Statements and their processing provenance in a believable manner.

> If trust can be put into the operations that record Signed Statements in a secure, append-only Registry via online operations, the same trust can be put into a corresponding Receipt that is the resulting documentation (i.e transparent signed statement, via a believable notarization function) issued by the Transparency Services and that can be validated in offline operations.

Reviewer: can we remove "believable" as it opens a bunch of questions for believable by whom and such. 
```suggestion
If trust can be put into the operations that record Signed Statements in a secure, append-only Registry via online operations, the same trust can be put into a corresponding Receipt that is the resulting documentation (i.e transparent signed statement, via an endorsement from a notarization function) issued by the Transparency Services and that can be validated in offline operations.
```
